.changelog/4553.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Enhanced the `sdk-changelog` script to include both the version number and release date when generating changelogs for SDK releases.
```

Comment:
Please use one of the approved templates in [Changelogs](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs).